apps/meteor/app/autotranslate/client/lib/autotranslate.ts

```diff
@@ -177,7 +177,8 @@ export const createAutoTranslateMessageStreamHandler = (): ((message: ITranslate
 				subscription &&
 				subscription.autoTranslate === true &&
 				message.msg &&
-				(!message.translations || !message.translations[language])
+				(!message.translations ||
+					(!message.translations[language] && !message.attachments?.some((attachment) => attachment?.translations?.[language])))
```

Comment:
I see this `message.attachments.some` pattern appear a lot of times throughout the code. I added a comment about the `ITranslatedMessageAttachment` and I think a helper function should be made to test this cases.